.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
@Andraquin I was thinking we will have a preset similar to  this ` "presets": ["es2015", "react","stage-0"]`, does your solution keep in mind we will transpile React `.jsx` files also

Reply:
@aanchirinah No, I didn't configure babel in view of React, since the repo was named "apollo-ah-backend", I was thinking that another repo will be created for "apollo-ah-frontend"